lucastle6969/AntennaPod390
# [#200] Log analysis

[Story : #170]
[Task : #200]

The script : 
 - Summarizes and analyzes Unit tests
  - Summarizes and analyzes Linting reports
  - Summarizes and analyzes Coverage reports
  - Writes output to file with the time and day it was generated in the "LogAnalysis" folder

To test: 
Step 1. Run the following scripts : 
gradlew testFreeDebugUnitTest
gradlew lintFreeDebug
gradlew testFreeDebugUnitTestCoverage

Step 2. Run logAnalysisScript.py 

Step 3. Verify that a text file was generated in the LogAnalysis folder and has the appropriate information
